src/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+const ncNews = "https://oscar-northcoders-news.onrender.com";
+
+export const getArticles = () => {
+  const articlesURL = ncNews + "/api/articles";
+  return axios.get(articlesURL).then((response) => {
+    return response.data?.articles;
+  });
+};
+
+export const getSingleArticle = (articleId) => {
+  console.log("fetching article id: ", articleId);
+  const singleArticleURL = `${ncNews}/api/articles/${articleId}`;
+  return axios.get(singleArticleURL).then((response) => {
+    console.log(response);
+    return response.data?.article;
+  });
+};
+
+export const getComments = (articleId) => {
+  const articleCommentsURL = `${ncNews}/api/articles/${articleId}/comments`;
+  return axios.get(articleCommentsURL).then((response) => {
+    console.log(response);
+    return response.data?.comments;
```

Comment:
Why have you made this conditional? You built the server, so you know what key your data will be returned on.